### ISSUE TITLE
Consensus

### ISSUE THREAD

**Issue by PrasRangarajan** [maher243/BlockSim#6]
Is it possible to edit the consensus algorithm and write in your own for testing ?

**Comment by muntaserabd1** [comment]
I am interested in this issue.

**Comment by ercangunbilek** [comment]
Up

**Comment by maher243** [comment]
Yes of course, model=1 and model=2 implements PoW, while model=3 implement POA. you can extend model=0 to implement any consensus algorithm.

**Comment by Mahmoudcnxz** [comment]
السلام عليكم 
دكتور ماهر الحربي تحية طيبه 
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور طلب من عندي تطوير وتحسين الخوارزميه pow  وتحديد مواصفات البروتوكول وتحليل النتائج وكذالك طلب من عندي الدكتور اطلع نفس النتائج الموجوده في الموقع في figure 5 ...... كيف يمكنني التوصل إلى نفس النتائج

**Comment by Mahmoudcnxz** [comment]
السلام عليكم
دكتور ماهر الحربي تحية طيبه
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور طلب من عندي تطوير وتحسين الخوارزميه pow و تحديد مواصفات البروتوكول وكذالك طلب من عندي تستخرج نفس النتائج الموجود في الموقع في الشكل figure 5...... كيف يمكن أن أحصل على  نفس النتائج

**Comment by Mahmoudcnxz** [comment]
السلام عليكم
دكتور ماهر الحربي تحية طيبه
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور طلب من عندي  تطوير وتحسين الخوارزميه pow و تحديد مواصفات وكذالك طلب من عندي تستخرج نفس النتائج الموجود في الموقع في الشكل 5 ...... كيف يمكن أن أحصل على نفس النتائج

**Comment by Mahmoudcnxz** [comment]
السلام عليكم
دكتور ماهر الحربي تحية طيبه
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور طلب من عندي تطوير وتحسين الخوارزميه pow و تحديد مواصفات البروتوكول وكذالك طلب من عندي إيجاد  نفس النتائج في الموقع في الشكل 5 ...... كيف يمكن أن أحصل على نفس النتائج وشكرا

**Comment by Mahmoudcnxz** [comment]
السلام عليكم
دكتور ماهر الحربي تحية طيبه
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور طلب من عندي  تطوير وتحسين الخوارزميه pow و تحديد مواصفات البروتوكول وكذالك طلب النتائج نفس النتائج في الموقع في الشكل 5

**Comment by Mahmoudcnxz** [comment]
السلام عليكم
دكتور ماهر الحربي تحية طيبه
أنا عندي بحث دكتوراه في جامعة جورج الأمريكيه .... كان موضوع البحث هو BlockSim الي رافعه حضرتك على github ....... الدكتور قال لي اعمل  تطوير وتحسين الخوارزميه pow و تحديد مواصفات بروتوكول وكذالك طلب النتائج نفس النتائج في الموقع في الشكل 5
